pycbc/waveform/parameters.py

```diff
@@ -417,32 +417,34 @@ def docstr(self, prefix='', include_label=True):
                 dtype=float, default=0., label=r"$\delta$",
                 description="Mean anomaly of the periastron (rad).")
 tc = Parameter("tc",
-                dtype=float, default=None, label=r"$t_c$ (s)",
-                description="Coalescence time (s).")
+               dtype=float, default=None, label=r"$t_c$ (s)",
+               description="Coalescence time (s) in "
+                           "geocentric/SSB/LISA coords.")
 delta_tc = Parameter("delta_tc", dtype=float,
                      label=r"$\Delta t_c~(\rm{s})$",
-                     description="Coalesence time offset.")
+                     description="Coalesence time offset in " +
+                     "geocentric/SSB/LISA coords.")
 ra = Parameter("ra",
-                dtype=float, default=0., label=r"$\alpha$",
-                description="Right ascension (rad).")
+               dtype=float, default=0., label=r"$\alpha$",
+               description="Right ascension (rad) in geocentric coords.")
```

Comment:
It doesn't make sense to say that ra/dec are in geocentric coordinates. They are uniquely defined, so there is no extra coordinate freedom.